### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by matthewmcgarvey**
Some unused code I found while looking around the codebase.

### CODE REVIEW

File: src/invidious/comments.cr [hunk 1]
```diff
@@ -60,8 +60,6 @@ def fetch_youtube_comments(id, cursor, format, locale, thin_mode, region, sort_b
   case cursor
   when nil, ""
     ctoken = produce_comment_continuation(id, cursor: "", sort_by: sort_by)
-    # when .starts_with? "Ug"
-    #   ctoken = produce_comment_reply_continuation(id, video.ucid, cursor)
```
Comment (matthewmcgarvey) [inline src/invidious/comments.cr:64]: This has been commented out for two years https://github.com/iv-org/invidious/commit/7b53b6bfeff1670c1c5d3dd299b7ed6162a5e3c0

File: src/invidious/trending.cr [hunk 2]
```diff
@@ -20,13 +20,3 @@ def fetch_trending(trending_type, region, locale)
 
   return {trending, plid}
 end
-
-def extract_plid(url)
```
Comment (matthewmcgarvey) [inline src/invidious/trending.cr:24]: Last usage removed 5 months ago https://github.com/iv-org/invidious/commit/50267a6dd60d8bb95c477e33e4824f8b19bc6424

**Comment by SamantazFox** [comment]
> The StaticFileHandler code was confusing. Looks like it was unused from the initial commit but the message using the term "Patch" makes me think the intent was to monkey patch instead of making a new class? Maybe there's a way that it's actually in use and I'm missing it. [6cd8845](https://github.com/iv-org/invidious/commit/6cd884555c3ea3a108fe20d5b7ccbba9bd49bb4c)

Yeah, that's a "monkey patch", indeed.
And I'm going to be honnest, that's a part of the codebase that I don't know much. I've been mainly focusing on rewriting the more immediate (and broken) parts of the code, i.e YT scraping and a few end-user related things. Except in some cases, I've personally preferred to not touch what was working.

**Comment by SamantazFox** [comment]
So, I looked at the code, and the patch of the static file handler was to allow static files to be cached in memory, probably to improve performances by reducing disk I/O.

**Comment by matthewmcgarvey** [comment]
@SamantazFox right but the handler is never actually added to the the Kemal configuration or used anywhere.

**Comment by SamantazFox** [comment]
> @SamantazFox right but the handler is never actually added to the the Kemal configuration or used anywhere.

As you said, it's monkey patched. It overrides the following function: `Kemal::StaticFileHandler.call()`
https://github.com/kemalcr/kemal/blob/master/src/kemal/static_file_handler.cr

**Comment by matthewmcgarvey** [comment]
My bad, I read Kemal, but in my mind it didn't process that this isn't kemal 🙈

**Review by SamantazFox — APPROVED** [review]

**Comment by TheFrenchGhosty** [comment]
@matthewmcgarvey I know it's completely off topic: but glad to see you back :)

**Pull request merged**